libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -99,45 +99,77 @@ export class FluidTab extends LitElement {
     return this._active;
   }
   set active(active: boolean) {
+    const oldActive = this._active;
     // Only set active true if not disabled
     this._active = this.disabled === false ? active : false;
-    // TODO: Figure out why using the old value does not work. The attribute vanishes in the dom/is not set
-    this.requestUpdate('active');
+    this.requestUpdate('active', oldActive);
     this.tabindex = this.active ? 0 : -1;
   }
   private _active = false;
 
-  /** Dispatches the custom event  */
+  /** Defines whether the user focused an element by tabbing or not */
+  @property({ type: Boolean, reflect: false })
+  get tabbed(): boolean {
+    return this._tabbed;
+  }
+  set tabbed(tabbed: boolean) {
+    const oldTabbed = this.tabbed;
+    this._tabbed = tabbed;
+    this.requestUpdate('tabbed', oldTabbed);
+    this.tabindex = tabbed === true ? 0 : -1;
+  }
+  private _tabbed = false;
```

Comment:
Nitpick. 
Upon reading this, it was very confusing to have the name of the setter / member be the same as the function parameter. 
Nothing against it, but I think this is easier to grasp.
```suggestion
  set tabbed(value: boolean) {
    const oldTabbed = this.tabbed;
    this._tabbed = value;
    this.requestUpdate('tabbed', oldTabbed);
    this.tabindex = value === true ? 0 : -1;
  }
  private _tabbed = false;
```